ui/src/app/table/skills-library-table/size-pagination/size-pagination.component.ts

```diff
@@ -0,0 +1,29 @@
+import {Component, EventEmitter, Input, OnInit, Output} from "@angular/core"
+import {FormControl} from "@angular/forms"
+
+@Component({
+  selector: "app-size-pagination",
+  templateUrl: "./size-pagination.component.html",
+  styleUrls: ["./size-pagination.component.scss"]
+})
+export class SizePaginationComponent implements OnInit {
+
+  values: number[] = [50, 100, 150]
```

Comment:
Technically, this should be read only:
`readonly values: number[] = [50, 100, 150]`
if not typed as an enumeration. Similarly, if not an enum the input to onValueChanged() should be checked/validated against the values array. As it stands you can inject values from the browser to paginate on any number through this new component. Just an FYI, not required to change it for this feature but important to be aware of in general.

Reply:
readonly has been added, thanks David